test/testdata/exhaustive_default.go

```diff
@@ -0,0 +1,25 @@
+//args: -Eexhaustive
+//config_path: testdata/configs/exhaustive.yml
+package testdata
+
+type Direction int
+
+const (
+	North Direction = iota
+	East
+	South
+	West
+)
+
+func processDirection(d Direction) {
+	switch d { // ERROR "missing cases in switch of type Direction: East, West"
```

Comment:
Needed a testcase in this file that writes _something_ to stderr. Otherwise these lines in `runGoErrchk()` in linters_test.go fail with the following error:

```
An error is expected but got nil.
```

```go
func runGoErrchk(c *exec.Cmd, files []string, t *testing.T) {
	output, err := c.CombinedOutput()
	assert.Error(t, err)
	...
}
```

Reply:
I adjusted `runGoErrchk()` to account for this instead of having this extra "dummy" test case.